### PR TITLE
NP-849: Add instruction for configuring OVN-K log level

### PR DESCRIPTION
**Pull request by pliurh**
**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

File: docs/contributor/network/howto_ovn_loglevel.md
Comment (ggiguash) [inline docs/contributor/network/howto_ovn_loglevel.md:1]: I'm ready to approve this, but I wonder if we should move this information to https://github.com/openshift/microshift/blob/main/docs/user/debugging_tips.md?

Comment (pacevedom) [reply docs/contributor/network/howto_ovn_loglevel.md:1]: Or even some reference here: https://github.com/openshift/microshift/blob/main/docs/user/howto_config.md

Other than that, looks good to me.

Comment (pliurh) [reply docs/contributor/network/howto_ovn_loglevel.md:1]: Let me move this to the debugging tips.

File: docs/user/debugging_tips.md [hunk 1]
```diff
@@ -109,3 +109,31 @@ OpenShift result in pod security admission errors when run on
 MicroShift. Refer to the [pod security HowTo](howto_pod_security.md)
 for a detailed example of configuring a workload to run with a custom
 security context.
+
+## Configure Log Verbosity for OVN-Kubernetes
+
+The default log verbosity level in Microshift is set to `4` for OVN-Kubernetes
```
Comment (ggiguash) [inline docs/user/debugging_tips.md:115]:
```suggestion
The default log verbosity level in MicroShift is set to `4` for OVN-Kubernetes
```

File: docs/user/debugging_tips.md [hunk 2]
```diff
@@ -109,3 +109,31 @@ OpenShift result in pod security admission errors when run on
...
+and `info` for OVN. You have the option to configure the log verbosity for
+debugging and troubleshooting purposes. To do so, you can create a ConfigMap
+named `env-overrides` with specific keys in the namespace
+`openshift-ovn-kubernetes`, then restart the corresponding OVN-Kubernetes pods.
```
Comment (ggiguash) [inline docs/user/debugging_tips.md:119]:
```suggestion
named `env-overrides` with specific keys in the `openshift-ovn-kubernetes`
namespace and restart the corresponding OVN-Kubernetes pods. 
```

File: docs/user/debugging_tips.md [hunk 3]
```diff
@@ -109,3 +109,31 @@ OpenShift result in pod security admission errors when run on
...
+  annotations:
+data:
+  # to set the log levels for the ovnkube-node pod
+  # replace this with the node's name (from oc get nodes)
+  ip-10-0-135-96.us-east-2.compute.internal: |
+    OVN_LOG_LEVEL=dbg
+  # to adjust the log levels for the ovnkube-master pod, use _master
+  _master: |
+    OVN_KUBE_LOG_LEVEL=5
+    OVN_LOG_LEVEL=dbg
```
Comment (ggiguash) [inline docs/user/debugging_tips.md:138]: Can we add a reference in the comments here on where to obtain the possible values and their meaning?

**Review by ggiguash — CHANGES_REQUESTED** [review]

File: docs/contributor/network/howto_ovn_loglevel.md
Comment (ggiguash) [reply docs/contributor/network/howto_ovn_loglevel.md:1]: I added a few minor comments, LGTM otherwise.

File: docs/user/debugging_tips.md (hunk 3 above)
Comment (pliurh) [reply docs/user/debugging_tips.md:144]: added.

**Pull request merged**